### PR TITLE
Release/0.21.6

### PR DESCRIPTION
**Pull request by AlexWellsHS**
### 0.21.6

* Removed Frank related code and tests #1378
* Updated project for blue ocean CI #1379 
* Update RunLoop::SimControl references to RunLoop::Simctl

### TODO

- [x] Update version to 0.21.6 in [podspec](https://github.com/calabash/calabash-ios/blob/develop/Calabash.podspec#L5)
- [x] Replace references to RunLoop::SimControl with some other RunLoop method.

### CODE REVIEW

**Comment by jmoody** [comment]
Jenkins test this please

File: Calabash.podspec [hunk 1]
```diff
@@ -2,7 +2,7 @@
 Pod::Spec.new do |s|
 
   s.name         = 'Calabash'
-  s.version      = '0.21.4'
+  s.version      = '0.21.5'
```
Comment (jmoody) [inline Calabash.podspec]: Update to 0.21.6

**Comment by AlexWellsHS** [comment]
@jmoody not sure about commit f72a770. running `rbenv exec bundle exec rspec spec/lib spec/bin spec/integration` passes every test besides `calabash_sim_accessibility`. any advice would be much appreciated.

**Comment by jmoody** [comment]
> def calabash_sim_accessibility
> detect_accessibility_support

Delete these methods and any callers.

There are no replacements and enabling accessibility is no longer necessary.

As you have noticed, the CLI is not a good shape.

**Comment by AlexWellsHS** [comment]
@jmoody all tests passing. am i able to kick off jenkins myself?

**Comment by AlexWellsHS** [comment]
Jenkins test this please

**Comment by jmoody** [comment]
@AlexWellsHS 

The Jenkins tests have not run automatically because of some permissions problems.

The "green check box" indicates that you have signed the MSFT OSS agreement.

I have added you as a contributor.  This should allow Jenkins to run the tests automatically.

**Comment by AlexWellsHS** [comment]
Jenkins test this please

**Comment by AlexWellsHS** [comment]
@jmoody shall i follow the [Release Docs](https://github.com/calabash/calabash-ios/blob/develop/DEVELOPER_HOWTO.md#releasing) from here?

**Comment by jmoody** [comment]
@AlexWellsHS Those docs look out of date.  I will handle from here.

**Comment by AlexWellsHS** [comment]
@jmoody cant i just make a PR from master to develop since this was merged into master?
also, you told me to remind you to update the readme.md after the release.

**Comment by jmoody** [comment]
You must merge release/0.21.6 into _develop_.

> remind me

Thanks!

**Pull request merged**